project/package.scala

```diff
@@ -0,0 +1,7 @@
+package object akka {
+
+  implicit class RichBoolean(val b: Boolean) extends AnyVal {
+    final def apply[A](a: => A): Option[A] = if (b) Some(a) else None
+  }
```

Comment:
I also commented like that :-) It's from the original PR I added my changes on top: https://github.com/akka/akka/pull/17532#discussion_r30734311